src/WarcraftLegacies.Source/Setup/Spells/QuelthalasSpellSetup.cs

```diff
@@ -60,7 +60,7 @@ public static void Setup()
         OrbitRadius = 350,
         OrbitalPeriod = 4,
         OrbEffectPath = @"war3mapImported\OrbFireX.mdx",
-        Damage = new LeveledAbilityField<float> { Base = 25, PerLevel = 25 },
+        Damage = new LeveledAbilityField<float> { Base = 20, PerLevel = 25 },
```

Comment:
I feel like there should be a corresponding tooltip change for this.

Reply:
Updated